src/ResourceManager/EventHub/Commands.EventHub/Models/PSOperationAttributes.cs

```diff
@@ -0,0 +1,63 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.EventHub.Models
+{
+    using Azure.Management.EventHub.Models;
+
+    /// <summary>
+    /// A ServiceBus REST API operation
+    /// </summary>
+    public partial class PSOperationAttributes
```

Comment:
Is this ever used? If not, please delete.

Reply:
Resolved